### PR TITLE
Feature/colang 2/llm chat interface

### PR DESCRIPTION
**Pull request by schuellc-nvidia**
Adds prompts for Colang 2 compatibility for most OpenAI and LLama models.

### CODE REVIEW

File: nemoguardrails/actions/v2_x/generation.py [hunk 1]
```diff
@@ -683,8 +683,8 @@ async def generate_value(
             value = value[:-1]
 
         # Remove variable name from the left if it appears in the result (GTP-4o):
-        if isinstance(prompt, str):
```
Comment (drazvan) [inline nemoguardrails/actions/v2_x/generation.py:686]: Let's keep this one in as well. Otherwise, we break support for non-chat models.

**Review by drazvan — APPROVED** [review]
Looks good :+1:. Only a small change requested.

**Comment by drazvan** [comment]
Make sure you fix the conflicts as well.

**Comment by schuellc-nvidia** [comment]
Its fixed
________________________________

**Pull request merged**